src/pages/BusinessBankAccount/NewPage.js

```diff
@@ -0,0 +1,202 @@
+                                    title={this.props.translate('bankAccount.connectManually')}
+                                    onPress={() => {
+                                        this.setState({bankAccountAddMethod: CONST.BANK_ACCOUNT.ADD_METHOD.MANUAL});
+                                    }}
+                                    shouldShowRightIcon
+                                />
+                                <View style={[styles.m5, styles.flexRow, styles.justifyContentBetween]}>
+                                    <TextLink href="https://use.expensify.com/privacy">
+                                        {this.props.translate('common.privacy')}
+                                    </TextLink>
+                                    <View style={[styles.flexRow, styles.alignItemsCenter]}>
+                                        <TextLink
+                                            // eslint-disable-next-line max-len
+                                            href="https://community.expensify.com/discussion/5677/deep-dive-how-expensify-protects-your-information/"
+                                        >
+                                            {this.props.translate('bankAccount.yourDataIsSecure')}
+                                        </TextLink>
+                                        <View style={[styles.ml1]}>
+                                            <Icon src={Lock} fill={colors.blue} />
+                                        </View>
+                                    </View>
+                                </View>
+                            </View>
+                        </>
+                    )}
+                    {this.state.bankAccountAddMethod === CONST.BANK_ACCOUNT.ADD_METHOD.PLAID && (
+                        <AddPlaidBankAccount
+                            text={this.props.translate('bankAccount.plaidBodyCopy')}
+                            onSubmit={(args) => {
+                                console.debug(args);
+                            }}
+                            onExitPlaid={() => {
+                                this.setState({bankAccountAddMethod: undefined});
+                            }}
+                        />
+                    )}
+                    {this.state.bankAccountAddMethod === CONST.BANK_ACCOUNT.ADD_METHOD.MANUAL && (
+                        <>
+                            <View style={[styles.m5, styles.flex1]}>
+                                <Text style={[styles.mb5]}>
+                                    {this.props.translate('bankAccount.checkHelpLine')}
+                                </Text>
+                                <Image
+                                    resizeMode="contain"
+                                    style={[styles.exampleCheckImage, styles.mb5]}
+                                    source={exampleCheckImage}
+                                />
+                                <TextInputWithLabel
```

Comment:
Followup of my comment from earlier, can't you just use vanilla TextInputs here? I'm probably missing something.

Reply:
I think this decision will make more sense once...

1. We add error handling + take advantage of the `errorText` this is just a scaffolding step
2. all the inputs get refactored and we use TextInputWithLabel as a base to build off of to create the text inputs that @shawnborton would prefer we build out eventually.